src/assets/i18n/uk.json

```diff
@@ -116,7 +116,7 @@
   "ACME DNS-Authenticator": "",
   "ACME Server Directory URI": "",
   "Actions for {device}": "",
-  "Activates iSCSI Extensions for RDMA (iSER) in TrueNAS, enabling the iSCSI protocol to directly transfer data into and out of SCSI memory buffers for improved performance. Enabling is limited to TrueNAS Enterprise-licensed systems and requires the system and network environment have Remote Direct Memory Access (RDMA)-capable hardware.": "",
+  "Activates iSCSI Extensions for RDMA (iSER) in TrueNAS, enabling   the iSCSI protocol to directly transfer data into and out of SCSI memory buffers for improved   performance. Enabling is limited to TrueNAS Enterprise-licensed systems and requires the system   and network environment have Remote Direct Memory Access (RDMA)-capable hardware.": "",
```

Comment:
Extra spaces added. In fr.json and others where it was translated, it's now not translated.